### PR TITLE
add a community membership guidelines

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**
/kind documentation
**What this PR does / why we need it**:
Provides criteria for what can be applied for as a member of the kurator community and what kind of circumstances would result in the revocation of kurator community membership.
**Which issue(s) this PR fixes**:
Fixes # #499

### CODE REVIEW

File: community/membership/community-membership.md
Comment (hzxuzhonghu) [inline community/membership/community-membership.md:1]: Would you put reviewer and maintainer under `/membership`?

Comment (LiZhenCheng9527) [reply community/membership/community-membership.md:1]: I will add it.

Comment (hzxuzhonghu) [reply community/membership/community-membership.md:1]: Sorry i should be more clear. We donot want a role per directory. Would you refer to kubeedge https://github.com/kubeedge/community/blob/master/community-membership.md

File: community/membership/community-membership.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+## Approver
+
+Approvers have shown good technical judgement in feature design/development in the past.
+Has overall knowledge of the project and features in the project.
+
+### Requirements
+
+- Deep understanding of the technical goals and direction of the project.
+- Deep understanding of the technical domain (specifically the language) of the project.
+- Approver for at least 2 months
```
Comment (hzxuzhonghu) [inline community/membership/community-membership.md:64]: ?

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**